src/eth/rpc/rpc_middleware.rs

```diff
@@ -103,7 +103,7 @@ impl<'a> RpcServiceT<'a> for RpcMiddleware {
             rpc_client = %client,
             rpc_id = %request.id,
             rpc_method = %method,
-            rpc_params = %to_json_value(&request.params),
+            rpc_params = %to_json_string_pretty(&request.params),
```

Comment:
Pretty string is formatted with multiple lines and will break logging.

Reply:
Good catch, I was supposed to call to_json_string and not to_json_string_pretty.